src/geo/map/torque-layer.js

```diff
@@ -56,6 +55,23 @@ var TorqueLayer = LayerModelBase.extend({
     }
   },
 
+  _hasTurboCarto: function () {
+    var currentCartoCSS = this.get('cartocss');
+    var shader = new carto.RendererJS().render(currentCartoCSS);
+    var layer = shader.findLayer({ name: '#layer' });
+
+    if (layer) {
+      var layerShader = layer.getStyle({ property: 1 }, { zoom: 10 });
+
+      var markerFill = layerShader['marker-fill'];
```

Comment:
Perhaps we could iterate over the properties of `layerShader` and see if any of them is a `ramp`?